### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -11453,13 +11459,18 @@ domension->dimension
 domensions->dimensions
 domian->domain
 domians->domains
+dominante->dominant, dominate,
+dominanted->dominated
+dominantes->dominants, dominates,
+dominantion->domination
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:11465]:
```text
dominanting->dominating
```
https://github.com/search?q=dominanting&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11474,6 +11485,9 @@ donwloaded->downloaded
 donwloading->downloading
 donwloads->downloads
 doocument->document
+doocumented->documented
+doocumenting->documenting
+doocuments->documents
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:11490]:
```text
doocumentation->documentation
doocumentations->documentations
doocumentary->documentary
doocumentaries->documentaries
```
https://github.com/search?q=doocumentation&type=code
https://github.com/search?q=doocumentations&type=code
https://github.com/search?q=doocumentary&type=code
https://github.com/search?q=doocumentaries&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
@vikivivi please review

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**